### PR TITLE
cloud: suport GCP Tokyo region for data migration

### PR DESCRIPTION
**Pull request by Frank945946**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Data migration : suports GCP Tokyo region

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ This feature helps you migrate your database and its ongoing changes to TiDB Clo
     - AWS Tokyo (ap-northeast-1)
     - AWS Frankfurt (eu-central-1)
     - AWS Seoul (ap-northeast-2)
+    - GCP Tokyo (asia-northeast1)
```
Comment (Frank945946) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:25]:
```suggestion
    - GCP Tokyo (asia-northeast1) 
```

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ summary: Learn how to migrate data from MySQL-compatible databases hosted in Ama
 
 # Migrate MySQL-Compatible Databases to TiDB Cloud Using Data Migration
 
-This document describes how to migrate data from a MySQL-compatible database on a cloud provider (Amazon Aurora MySQL or Amazon Relational Database Service (RDS)) or on-premises to TiDB Cloud using the Data Migration feature of the TiDB Cloud console.
+This document describes how to migrate data from a MySQL-compatible database on a cloud provider (Amazon Aurora MySQL or Amazon Relational Database Service (RDS),Google Cloud SQL for MySQL) or on-premises to TiDB Cloud using the Data Migration feature of the TiDB Cloud console.
```
Comment (qiancai) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:8]:
```suggestion
This document describes how to migrate data from a MySQL-compatible database on a cloud provider (Amazon Aurora MySQL, Amazon Relational Database Service (RDS), or Google Cloud SQL for MySQL) or on-premises to TiDB Cloud using the Data Migration feature of the TiDB Cloud console.
```

**Review by qiancai — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 
-This document describes how to migrate data from a MySQL-compatible database on a cloud provider (Amazon Aurora MySQL or Amazon Relational Database Service (RDS)) or on-premises to TiDB Cloud using the Data Migration feature of the TiDB Cloud console.
+This document describes how to migrate data from a MySQL-compatible database on a cloud provider (Amazon Aurora MySQL, Amazon Relational Database Service (RDS), or Google Cloud SQL for MySQL) or on-premises to TiDB Cloud using the Data Migration feature of the TiDB Cloud console.
 
-This feature helps you migrate your database and its ongoing changes to TiDB Cloud (either in the same region or cross regions). Compared with solutions that require tools such as Dumpling and TiDB Lightning, this feature is easier to use. You do not need to manually dump data from the source database and then import it to TiDB Cloud. Instead, you can migrate data directly from the source database to TiDB Cloud in one go.
+This feature helps you migrate your source databases' existing data and ongoing changes to TiDB Cloud (either in the same region or cross regions)directly in one go.
```
Comment (hfxsd) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:10]:
```suggestion
This feature helps you migrate your source databases' existing data and ongoing changes to TiDB Cloud (either in the same region or cross regions) directly in one go. 
```

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 4]
```diff
@@ -105,29 +107,46 @@ Before creating a migration job, set up the network connection according to your
 
 - If you use AWS PrivateLink, set it up according to [Set Up Private Endpoint Connections](/tidb-cloud/set-up-private-endpoint-connections.md).
 
-- If you use VPC Peering, see the following instructions to configure the network.
+- If you use AWS VPC Peering, see the following instructions to configure the network.
 
-<details>
-<summary> Set up VPC Peering</summary>
+    <details>
```
Comment (hfxsd) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:112]: <details> 部分不能缩进，否则发布的时候会出问题。需要顶格。

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 5]
```diff
@@ -172,24 +191,24 @@ On the **Create Migration Job** page, configure the source and target connection
...
 ## Step 3: Choose the objects to be migrated
 
-1. Choose full data migration, incremental data migration, or both by choosing the checkboxes.
+1. Choose existing data migration, incremental data migration, or both by choosing the checkboxes.
 
     > **Tip:**
     >
-    > - To migrate data to TiDB Cloud once and for all, choose both **Full data migration** and **Incremental data migration**, which ensures data consistency between the source and target databases.
-    > - To migrate only the existing data of the source database to TiDB Cloud, only choose the **Full data migration** checkbox.
+    > - To migrate data to TiDB Cloud once and for all, choose both **existing data migration** and **Incremental data migration**, which ensures data consistency between the source and target databases.
```
Comment (qiancai) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:202]: 界面上显示的是 Existing data migration 吗

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 6]
```diff
@@ -172,24 +191,24 @@ On the **Create Migration Job** page, configure the source and target connection
...
+    > - To migrate only the existing data of the source database to TiDB Cloud, only choose the **existing data migration** checkbox.
```
Comment (qiancai) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:203]: 同上

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 7]
```diff
@@ -172,24 +191,24 @@ On the **Create Migration Job** page, configure the source and target connection
...
 
 2. On the **Choose Objects to Migrate** page, select the objects to be migrated. You can click **All** to select all objects, or click **Customize** and then click the checkbox next to the object name to select the object.
 
-    - If you click **All**, the migration job will migrate the existing data from the whole source database instance to TiDB Cloud and replicate ongoing changes after the full migration. Note that it happens only if you have selected the **Full data migration** and **Incremental data migration** checkboxes in the previous step.
+    - If you click **All**, the migration job will migrate the existing data from the whole source database instance to TiDB Cloud and replicate ongoing changes after the full migration. Note that it happens only if you have selected the **existing data migration** and **Incremental data migration** checkboxes in the previous step.
```
Comment (qiancai) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:207]: 同上

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 8]
```diff
@@ -172,24 +191,24 @@ On the **Create Migration Job** page, configure the source and target connection
...
 
         <img src="https://download.pingcap.com/images/docs/tidb-cloud/migration-job-select-all.png" width="60%" />
 
-    - If you click **Customize** and select some databases, the migration job will migrate the existing data and replicate ongoing changes of the selected databases to TiDB Cloud. Note that it happens only if you have selected the **Full data migration** and **Incremental data migration** checkboxes in the previous step.
+    - If you click **Customize** and select some databases, the migration job will migrate the existing data and replicate ongoing changes of the selected databases to TiDB Cloud. Note that it happens only if you have selected the **existing data migration** and **Incremental data migration** checkboxes in the previous step.
```
Comment (qiancai) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:211]: 同上

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 9]
```diff
@@ -105,29 +107,46 @@ Before creating a migration job, set up the network connection according to your
...
+        2. Enable the **Accepter DNS resolution** option.
 
-</details>
+    </details>
 
-### Enable binlogs
+- If you use GCP VPC Network Peering, see the following instructions to configure the network.
 
-To perform incremental data migration, make sure you have enabled binlogs of the upstream database, and the binlogs have been kept for more than 24 hours.
+    <details>
```
Comment (qiancai) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:132]: `<details>` 缩进问题，同上

**Review by D3Hunter — APPROVED** [review]
lgtm

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 10]
```diff
@@ -125,9 +127,24 @@ If your MySQL service is in an AWS VPC, take the following steps:
...
+
+If your MySQL service is in an GCP VPC, take the following steps:
+
+1. If it is a self-hosted mysql, please skip this step and go to the second step.If your MySQL service is Google Cloud SQL, you must expose a MySQL endpoint in the associated VPC of the Google Cloud SQL instance. You may need to use the [Cloud SQL Auth proxy](https://cloud.google.com/sql/docs/mysql/sql-proxy) which is developed by Google.
+
+2. [Set up a VPC peering connection](https://docs.pingcap.com/tidbcloud/set-up-vpc-peering-connections) between the VPC of the MySQL service and your TiDB cluster.
+
+3. Modify the ingress firewall rules of the VPC where MySQL is located.
+
+    You must add [the CIDR of the region where your TiDB Cloud cluster is located](https://docs.pingcap.com/tidbcloud/set-up-vpc-peering-connections#prerequisite-set-a-project-cidr) to the ingress firewall rules. Doing so allows the traffic to flow from your TiDB Cluster to the MySQL endpoint.
```
Comment (qiancai) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:141]:
```suggestion
    You must add [the CIDR of the region where your TiDB Cloud cluster is located](/tidb-cloud/set-up-vpc-peering-connections.md#prerequisite-set-a-project-cidr) to the ingress firewall rules, which allows the traffic to flow from your TiDB cluster to the MySQL endpoint. 
```

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 11]
```diff
@@ -55,6 +56,7 @@ Data Migration supports the following data sources and versions:
 - MySQL 5.6, 5.7, and 8.0 local instances or on a public cloud provider. Note that MySQL 8.0 is still experimental on TiDB Cloud and might have incompatibility issues.
 - Amazon Aurora (MySQL 5.6 and 5.7)
 - Amazon RDS (MySQL 5.7)
+- Google Cloud SQL for (MySQL 5.6 and 5.7)
```
Comment (qiancai) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:59]:
```suggestion
- Google Cloud SQL for MySQL 5.6 and 5.7
```

**Review by hfxsd — APPROVED** [review]

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 12]
```diff
@@ -172,24 +189,24 @@ On the **Create Migration Job** page, configure the source and target connection
 5. Take action according to the message you see:
 
     - If you use Public IP or VPC Peering, you need to add the Data Migration service's IP addresses to the IP Access List of your source database and firewall (if any).
-    - If you use Private Link, you are prompted to accept the endpoint request. Go to the [AWS VPC console](https://us-west-2.console.aws.amazon.com/vpc/home), and click **Endpoint services** to accept the endpoint request.
+    - If you use AWS Private Link, you are prompted to accept the endpoint request. Go to the [AWS VPC console](https://us-west-2.console.aws.amazon.com/vpc/home), and click **Endpoint services** to accept the endpoint request.
```
Comment (hfxsd) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:192]: 用 GCP 有类似的问题吗？

Comment (Frank945946) [reply tidb-cloud/migrate-from-mysql-using-data-migration.md:192]: GCP 还不支持 Private link 所以暂时没有这个问题

**Pull request merged**